tools/evmc/main.cpp

```diff
@@ -20,14 +76,53 @@ int main(int argc, const char** argv)
             return 0;
         }
 
+        if (run_cmd)
+        {
+            auto code = evmc::from_hex(code_hex);
+
+            std::cout << "Running " << vm_config << "\n";
+            evmc_loader_error_code ec;
+            auto vm = evmc::VM{evmc_load_and_configure(vm_config.c_str(), &ec)};
+            if (ec != EVMC_LOADER_SUCCESS)
+            {
+                const auto error = evmc_last_error_msg();
+                if (error != nullptr)
+                    std::cerr << error << "\n";
+                else
+                    std::cerr << "Loading error " << ec << "\n";
+                return static_cast<int>(ec);
+            }
+
+            evmc::MockedHost host;
+            evmc_message msg{};
+            msg.gas = 10000000;
+
+            auto result = vm.execute(host, EVMC_ISTANBUL, msg, code.data(), code.size());
```

Comment:
Maybe an output message here for: `"Executing on Istanbul with " << msg.gas << "gas"`